interface/patient_file/encounter/forms.php

```diff
@@ -667,10 +667,18 @@ function myGetRegistered($state="1", $limit="unlimited", $offset="0") {
     if ($result['squad'] && ! acl_check('squads', $result['squad'])) {
         $pass_sens_squad = false;
     }
+
+    //fetch acl for category of given encounter
+    $pc_catid = fetchCategoryIdByEncounter($encounter);
+    $postCalendarCategoryACO = fetchPostCalendarCategoryACO($pc_catid);
+    $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
```

Comment:
actually changed it back to 2 lines so can take care of case where no aco is set for cateogry